etl-job/fhir_import_export.py

```diff
@@ -319,22 +352,28 @@ def _put(input_data, output, program, project, user):
     # check permissions
     can_create = _can_create(output, program, user)
     output['logs'].append(f"CAN CREATE: {can_create}")
-    file_path = f"/root/studies/{project}/"
-    if can_create:
-        object_id = _get_object_id(input_data)
-        if object_id:
-            # get the meta data file
-            if _download_and_unzip(object_id, file_path, output):
-
-                # tell user what files were found
-                for _ in pathlib.Path(file_path).glob('*'):
-                    output['files'].append(str(_))
-
-                # load the study into the database and elastic search
-                _load_all(project, f"{program}-{project}", output)
-
-        else:
-            output['logs'].append("OBJECT ID NOT FOUND")
+    assert can_create, f"No create permissions on {program}"
+    assert 'push' in input_data, "input data must contain a `push`"
+    for commit in input_data['push']['commits']:
+        assert 'object_id' in commit, "commit must contain an `object_id`"
+        object_id = commit['object_id']
+        assert object_id, "object_id must not be empty"
+        assert 'commit_id' in commit, "commit must contain a `commit_id`"
```

Comment:
Assert statements in the older code append outputs to logs while they are in the ETL pod, while assert statements in this new PR output to console. Shouldn't these be added to the output logs instead?